netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -58,14 +60,30 @@
     private static final Pattern NEW_LINE = Pattern.compile(
         "^ {0,3}(\\S|(\\S.*\\S)) ?$", Pattern.MULTILINE
     );
+    /**
+     * Html &lt;pre&gt;&ltcode&gt; fragment.
+     */
+    private static final String PRE_CODE = "<pre><code>";
+    /**
+     * Html &lt;/code&gt;&lt/end&gt; fragment.
+     */
+    private static final String END_CODE_PRE = "</code></pre>";
```

Comment:
@danff this constant is used only in one method, please make it a variable there